### PR TITLE
Feature: Rails Budget App

### PR DESCRIPTION
**Pull request by zainsadaqat**
# In this pull request, I've done the following:

- Splash screen

- [ ] A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.

- [ ] Sign up and log in pages

- [ ] The user should be able to register in the app with full name, email and password (all mandatory).

- [ ] The user can log into the app using email and password.

- [ ] If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).

- Home page (catagories page)

- [ ] When the user logs in, they are presented with the categories page.

- [ ] For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.

- [ ] When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.

- [ ] There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

- Transactions page

- [ ] For a given category, the list of transactions is presented, ordered by the most recent.

- [ ] At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).

- [ ] There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.

- [ ] When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new category" page

- [ ] The user fills out a form to create a new category, indicating their name and icon (both mandatory).

- [ ] The user click (or taps) the "Save" button to create the new category, and is taken to the home page on success.

- [ ] When the user clicks on the "Back" button (<), the user navigates to the home page.

### CODE REVIEW

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @zainsadaqat, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Correct Gitflow.
- Use devise for authentication.

## Required Changes ♻️
- [x] Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
   - a description of the project.
   - a demo of the project features.
    - you should also highlight some interesting piece of code or something you built that you are very proud of.
   
 - [x] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.
 
 - [x] Kindly add a README file that gives a full description of your project.
 
 - [x] Kindly deploy the application using Heroku, Netlify, or GitHub Pages, the project should be deployed properly, the home and detail page works as expectedand a link to the deployed project is present in the README.
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zainsadaqat** [comment]
Hey @bimbolabuari 
Can you please review my project? Thanks!

File: README.md [hunk 1]
```diff
@@ -1,24 +1,68 @@
...
+- RSPEC-RAILS, Capybara
 
-* Configuration
+### Unfortunately, my LOOM is not working. I'm trying this for a couple of hours.
 
-* Database creation
+## Live Demo
 
-* Database initialization
+[Live Demo Link](https://livedemo.com)
```
Comment (bimbolabuari) [inline README.md:16]: [ Optional] This is part of the project requirement, Kindly add your deployed link here 🙂

Comment (bimbolabuari) [inline README.md:16]: [ Optional] This is part of the project requirement, Kindly add your deployed link here 🙂

File: README.md [hunk 2]
```diff
@@ -1,24 +1,68 @@
...
+rails s
+```
+
+## Authors
+
+👤 **Zain Sadaqat**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (bimbolabuari) [inline README.md:50]: [Optional] Kindly add your social handles here 👍

File: README.md [hunk 3]
```diff
@@ -1,24 +1,68 @@
...
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- RSPEC-RAILS, Capybara
 
-* Configuration
+### Unfortunately, my LOOM is not working. I'm trying this for a couple of hours.
```
Comment (bimbolabuari) [inline README.md:12]: Please fix this before merging ✊

**Review by bimbolabuari — APPROVED** [review]
Hi @zainsadaqat, 

Your project is complete!  Kindly fix the optional changes than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (zainsadaqat) [reply README.md:12]: I'm trying for more than 2 hours but it's not working. I un-installed and re-installed ti again but still same.

File: README.md (hunk 2 above)
Comment (zainsadaqat) [reply README.md:50]: Yes, Sure!

File: README.md (hunk 1 above)
Comment (zainsadaqat) [reply README.md:16]: Yes, Sure!

Comment (zainsadaqat) [reply README.md:16]: Yes, Sure!

**Pull request merged**